test/sqllogictest/transform/predicate_pushdown.slt

```diff
@@ -394,6 +394,48 @@ Source mz_catalog.mz_materialized_views
 
 EOF
 
+# Regression tests for #17515
+# Push predicates through Threshold and TopK
```

Comment:
There is no TopK test in this file. (Which is ok, since there is one in `src/transform/tests/testdata/predicate-pushdown`, just the comment should be updated.)

Reply:
If you could add a SQL-based test for TopK that would be much appreciated.

Reply:
Done!